db/migrate/20221211063622_move_addresses_to_table.rb

```diff
@@ -0,0 +1,16 @@
+class MoveAddressesToTable < ActiveRecord::Migration[6.0]
+  def up
+    Order.find_each do |order|
+      order.shipping_address = Address.create!(address: order.address, zipcode: order.zipcode)
```

Comment:
In a production environment, these transitions would run plain SQL queries to create and update records for the `addresses` table, so the transition doesn't need to rely on model behavior, but I decided to skip that for the time being.